src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -167,7 +168,10 @@ export class HaConfigDeviceDashboard extends LitElement {
                 )
                 .join(", ")
             : "No integration",
-          battery_entity: this._batteryEntity(device.id, deviceEntityLookup),
+          battery_entity: [
```

Comment:
Update the `DeviceRowData` interface to:
`battery_entity: [string | undefined, string | undefined];`